internal/v1/api.yaml

```
@@ -574,6 +574,21 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/Filesystem'
+        users:
+          type: array
+          items:
+            $ref: '#/components/schemas/User'
+    User:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          example: "user1"
+        key:
+          type: string
+          example: "userkey"
```

Let's make this something that resembles an actual ssh key.

```suggestion
          example: "ssh-rsa AAAAB3NzaC1"
```